topics/10_projects/main_building_a_project.md

```diff
@@ -1,3 +1,52 @@
 # Topic 10: Building a Test Automation Suite, utilizing both web and API testing
 
-Coming soon.
+This project is your chance to bring together everything you’ve learned during the training. You’ll gain hands-on experience by building a Test Automation Suite that includes both Web UI and API testing. The goal is to help you apply your skills in a real-world scenario and showcase your knowledge as a Test Automation Engineer.
+
+
+## Project Requirements
+
+The project should implement a Test Automation Framework Architecture where you can develop a suite of tests with the following components:
+
+
+### 1. Web UI Test Suite
+
+Create a suite of tests for a web user interface. At least two functionalities should be included. Feel free to use any testing framework you prefer.
+
+
+### 2. API Test Suite
+
+Develop a suite of tests for an API. This should include tests for at least these API methods: GET, POST, PUT, DELETE. You can also include tests for Data validation, and error handling. Use pytest and requests library for API testing.
+
+
+### 3. Documentation
+
+Provide basic documentation for your project, including:
+- **Test Strategy**: Outline your overall approach to testing.
+- **Test Plan**: Detail the specific tests to be performed, the schedule, and resources.
+- **Test Cases**: Describe the individual tests in detail.
+- **Test Report**: Summarize the results of the tests, including any issues found and their severity.
+
+
+### 4. Optional Enhancements
+
+Consider adding enhancements like linters and formatters to your project. These tools can help maintain code quality by enforcing coding standards and formatting rules. They can also help identify potential issues early in the development process.
```

Comment:
Good point but imho better to request it explicitly like:

Code should pass the following code style checking:
e.g.
pycodestyle
flake8
black